src/main/java/cms/gov/madie/measure/utils/JsonUtil.java

```diff
@@ -332,9 +339,14 @@ private static void appendPopulationValues(
                 observation != null && observation.getDefinition() != null
                     ? PopulationType.fromCode(
                         observation
-                            .getDefinition() // e.g. Denominator Observations ->
+                            .getDefinition() // e.g. Denominator Observations or: Denominator
```

Comment:
are we sure if `observation.getDefinition` value is always Denominator/numerator Observations?

Reply:
@adongare Actually I'm not sure, because I found there was at at least two different values: Denominator Observation**s** and Denominator Observation

Reply:
I think we should verify once by naming the observation function otherthan "Denominator/Numerator Observations" once

Reply:
@adongare This is fixed, thanks a lot for pointing it out!